src/main/java/org/ibp/api/rest/study/StudyGermplasmResource.java

```diff
@@ -7,26 +7,46 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
 
 import javax.annotation.Resource;
+import java.util.List;
 
 @Api(value = "Study Germplasm Services")
 @Controller
 @RequestMapping("/crops")
 public class StudyGermplasmResource {
 
-    @Resource
-    private StudyGermplasmService studyGermplasmService;
+	@Resource
+	private StudyGermplasmService studyGermplasmService;
 
-    @ApiOperation(value = "Replace germplasm entry in study",
-            notes = "Replace germplasm entry in study")
-    @RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/germplasm/{entryId}", method = RequestMethod.PUT)
-    @ResponseBody
-    public ResponseEntity<StudyGermplasmDto> replaceStudyGermplasm(final @PathVariable String cropname, @PathVariable final String programUUID,
-                                                                   @PathVariable final Integer studyId, @PathVariable final Integer entryId, @RequestBody final StudyGermplasmDto studyGermplasmDto) {
-        return new ResponseEntity<>(this.studyGermplasmService.replaceStudyGermplasm(studyId, entryId, studyGermplasmDto),
-                HttpStatus.OK);
+	@ApiOperation(value = "Replace germplasm entry in study",
+		notes = "Replace germplasm entry in study")
+	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/germplasm/{entryId}", method = RequestMethod.PUT)
+	@ResponseBody
+	public ResponseEntity<StudyGermplasmDto> replaceStudyGermplasm(final @PathVariable String cropname,
+		@PathVariable final String programUUID,
+		@PathVariable final Integer studyId, @PathVariable final Integer entryId, @RequestBody final StudyGermplasmDto studyGermplasmDto) {
+		return new ResponseEntity<>(this.studyGermplasmService.replaceStudyGermplasm(studyId, entryId, studyGermplasmDto),
+			HttpStatus.OK);
+
+	}
+
+	@ApiOperation(value = "Create germplasm entries in study based on the specified germplasm list",
+		notes = "Create germplasm entries in study based on the specified germplasm list")
+	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/germplasm/list/{germplasmListId}", method = RequestMethod.POST)
```

Comment:
Hi @abatac 
I think the service should be named entry-lists and germplasmList should be included in the body